src/utl/test/test_report_metrics.tcl

```diff
@@ -0,0 +1,6 @@
+read_liberty ./../../../test/Nangate45/Nangate45_fast.lib
```

Comment:
I think your CI test should include corners

Reply:
Do I understand correctly that you mean that we need to call the report_metrics command with the include_erc and include_clock_skew flags?

Reply:
I mean that you need to define a couple of corners and load liberty files in for those
```
define_corners fast slow
read_liberty -corner fast  ./../../../test/Nangate45/Nangate45_fast.lib
read_liberty -corner slow ./../../../test/Nangate45/Nangate45_slow.lib
```
that should allow you to test the `sta::corners` code you have.

And yes, if the function has flags we should be testing that behavior as well.

Reply:
Hi @gadfort. I changed the test on your advice, and now I have two corners in the test:
```
define_corners fast slow
read_liberty -corner fast  ./../../../test/Nangate45/Nangate45_fast.lib
read_liberty -corner slow ./../../../test/Nangate45/Nangate45_slow.lib
read_lef ./../../../test/Nangate45/Nangate45.lef
read_verilog ./../../../test/gcd_nangate45.v
link_design gcd
read_sdc ./../../../test/timing_api_2.sdc
report_metrics 2 final false false
```
But I can't generate the ok file - I get error in the output:
```
[INFO ODB-0227] LEF file: ./../../../test/Nangate45/Nangate45.lef, created 22 layers, 27 vias, 135 library cells
Report metrics stage 2, final...
Metrics results directory: /home/dav77227/OpenROAD/results
[ERROR STA-0102] _f0aa93ee7b550000_p_Corner is not the name of process corner.
Error: test_report_metrics.tcl, 8 STA-0102
ERROR 
```
I suppose that the case has something incorrect. Could you please help to fix the test?

Reply:
see my comment above